pkgs/applications/misc/almanah/default.nix

```diff
@@ -66,7 +66,7 @@ stdenv.mkDerivation rec {
     ninja
     pkg-config
     python3
-    wrapGAppsHook
+    wrapGAppsHook4
```

Comment:
Almanah still uses gtk3.

Reply:
Oh, seems to be a mistake triggered by that evolution-data-server